src/ipfs-server.ts

```diff
@@ -36,7 +36,7 @@ const IPFS_ENABLE_GATEWAY = process.env.IPFS_ENABLE_GATEWAY ? process.env.IPFS_E
 
 const IPFS_DHT_SERVER_MODE = process.env.IPFS_DHT_SERVER_MODE === 'true'
```

Comment:
Does this need a default also?

Reply:
I guess default is `false` now?

Reply:
Yeah default is false. Not sure we need to change that behavior bc I don't know what that does but we can come back to that in another pr if needed.